src/python/serenity_types/pricing/derivatives/options/volsurface.py

```diff
@@ -162,11 +160,10 @@ class InterpolatedVolatilitySurface(BaseModel):
     All fitted vols, the z-axis in the mesh.
     """
 
-    option_types: List[OptionType]
```

Comment:
this means InterpolatedVolatilitySurface now supports both PUT and CALL?

Reply:
Yes. Per @youngsuklee-cloudwall and @kmgivens they are going to fit a surface from both option types rather than splitting into two, so this field is no longer needed.